Makefile

```diff
@@ -1,6 +1,25 @@
 CONTAINER_ENGINE := docker
 GO := go
 
+# Obtain the current system architecture.
```

Comment:
Because sysbox-runc unit tests don't necessarily use the top-level Makefile. They can be executed directly from within the sysbox-runc subrepo via `make test`.

Reply:
Got it -- haven't ever invoked this target directly from here.